### PR TITLE
[#152] Small usability enhancement for configuring an insecure port

### PR DESCRIPTION
**Pull request by sysexcontrol**
Signed-off-by: Karsten Frank <Karsten.Frank@bosch-si.com>

### CODE REVIEW

File: application/src/main/resources/application.yml [hunk 1]
```diff
@@ -10,7 +10,6 @@ hono:
     singleTenant: false
     insecurePortEnabled: false
     bindAddressInsecurePort: 127.0.0.1
```
Comment (sophokles73) [inline application/src/main/resources/application.yml]: why do we configure these two variables (`insecurePortEnabled` and `bindAddressInsecurePort``)?

Comment (sysexcontrol) [inline application/src/main/resources/application.yml]: Actually, we do not need to configure them. When the insecure port was still contained here, it was a convenient way to have all values together.
Now it seems better to completely remove them - WDYT?

Comment (sophokles73) [inline application/src/main/resources/application.yml]: I agree

**Comment by sysexcontrol** [comment]
provided as squashed commit already (due to the minor change)

**Pull request merged**